subscriptions/fixed-price/checkout/node/src/server.js

```diff
@@ -65,14 +72,16 @@ app.post('/create-checkout-session/', async (req, res) => {
 })
 
 // Subscriptions
-app.put('/upgrade-subscription/:id', async (req, res) => {
+app.put('/update-subscription/:id', async (req, res) => {
   const subscriptionId = req.params.id
+  const amount = req.body.amount // e.g. '20'
```

Comment:
We've decided not to do this and make the amount and biling_interval hardocded. The reason being is that this endpoint as it is unprotected and might lead the merchant to think that they might want to read amount and the bilingInterval for the body and have it as unprotected. We might end up giving a wrong example/leading users in a wrong direction by doing this.

Reply:
Fair enough I see the point, just pushed a fix reverting to upgrade endpoint.